src/material/list/list-option-types.ts

```diff
@@ -9,18 +9,27 @@
 import {InjectionToken} from '@angular/core';
 
 /**
- * Type describing possible positions of a checkbox in a list option
+ * Type describing possible positions of a checkbox or radio in a list option
  * with respect to the list item's text.
+ *
+ * @deprecated Use `MatListOptionTogglePosition` instead.
+ * @breaking-change 17.0.0
  */
 export type MatListOptionCheckboxPosition = 'before' | 'after';
 
+/**
+ * Type describing possible positions of a checkbox or radio in a list option
+ * with respect to the list item's text.
+ */
+export type MatListOptionTogglePosition = 'before' | 'after';
```

Comment:
We shouldn't redeclare the type like this. Instead we can rename it and re-export it under the new name. E.g.

```
// list-option-types.ts
export type MatListOptionTogglePosition = 'before' | 'after';

// public-api.ts
export {MatListOptionTogglePosition as MatListOptionCheckboxPosition}
```

Reply:
good suggestion, didn't think to do that. I ended up deleting `MatListOptionCheckboxPosition` since this is targeting RC